tutorial/LFRic/background/LFRic_PSy.md

```diff
@@ -0,0 +1,206 @@
+# LFRic PSy
+
+[*PSy layer in LFRic in PSyclone documentation*](
+https://psyclone.readthedocs.io/en/stable/dynamo0p3.html#psy-layer)
+
+## What PSy layer does
+
+As said in the [*What algorithms do* section](
+LFRic_algorithm.md#what-algorithms-do), LFRic algorithms perform
+mathematical operations on LFRic data objects and pass them as
+arguments to kernel and built-in calls in `invoke`s. As said in the
+[*What kernels do* section](LFRic_kernel.md#what-kernels-do), LFRic
+kernels perform mathematical operations on a subset of data points
+of these objects.
+
+In the *Parallel Systems* or, in short, [PSy layer](
+https://psyclone.readthedocs.io/en/stable/psy_layer.html) PSyclone
+generates calls to *proxies* of the LFRic data objects passed from the
+algorithm layer and accesses the object data by dereferencing the object
+proxies. The data and other required information is then passed to
+kernels as illustrated in the [*Kernel subroutine* section](
+LFRic_kernel.md#kernel-subroutine).
```

Comment:
This relates back to a long standing issue I've had. The things we call "proxies" aren't really that in the normal usage of the term. They are much more like "accessors". I don't know what you want to do about this because they are called proxies in the source.

Reply:
Good point. I rephrased this as "... PSyclone generates calls to accessor classes of the LFRic data objects passed from the algorithm layer, referred to as *proxies*, and accesses the object data by dereferencing the object proxies." Also rephrased in the algorithm section.